com.unity.multiplayer.mlapi/Tests/Editor/InternalMessageHandlerTests.cs

```diff
@@ -0,0 +1,387 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using MLAPI.Configuration;
+using MLAPI.Internal;
+using MLAPI.Messaging;
+using MLAPI.Messaging.Buffering;
+using MLAPI.Serialization;
+using MLAPI.Transports;
+using MLAPI.Transports.Tasks;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using Object = UnityEngine.Object;
+
+namespace MLAPI.EditorTests
+{
+    public class InternalMessageHandlerTests
```

Comment:
This is more of a networkmanager test than an internalmessagehandler test - maybe rename the file/class?